src/Config.zig

```diff
@@ -214,10 +217,6 @@ pub fn configChanged(config: *Config, allocator: std.mem.Allocator, builtin_crea
     }
 
     if (null == config.global_cache_path) {
```

Comment:
Might be best to just move this block above the one creating the build runner file instead of using `cache_dir_path`